plugins/auth-backend/api-report.md

```diff
@@ -81,33 +81,6 @@ export type AwsAlbResult = {
   accessToken: string;
 };
 
-// @public (undocumented)
-export type BitbucketOAuthResult = {
-  fullProfile: BitbucketPassportProfile;
-  params: {
-    id_token?: string;
-    scope: string;
-    expires_in: number;
-  };
-  accessToken: string;
-  refreshToken?: string;
-};
-
-// @public (undocumented)
-export type BitbucketPassportProfile = Profile & {
-  id?: string;
-  displayName?: string;
-  username?: string;
-  avatarUrl?: string;
-  _json?: {
-    links?: {
-      avatar?: {
-        href?: string;
-      };
-    };
-  };
-};
```

Comment:
Noting that these are fine to remove because they where internal types exported by mistake, they're not referenced anywhere.